src/main/groovy/com/cloudbees/groovy/cps/CpsTransformer.groovy

```diff
@@ -214,6 +205,48 @@ class CpsTransformer extends CompilationCustomizer implements GroovyCodeVisitor
     }
 
     /**
+     * Generates code that instantiates a new {@link Builder}.
+     *
+     * <p>
+     * Hook for subtypes to tweak builder, for example to {@link Builder#contextualize(com.cloudbees.groovy.cps.sandbox.CallSiteTag...)}
+     *
+     * <pre>
+     * Builder b = new Builder(new MethodLocation(...));
+     * b.withClosureType(...);
+     * </pre>
+     *
+     * @param m
+     *      Method being transformed.
+     */
+    protected Expression makeBuilder(MethodNode m) {
+        Expression b = new ConstructorCallExpression(BUIDER_TYPE, new TupleExpression(
+                new ConstructorCallExpression(METHOD_LOCATION_TYPE, new TupleExpression(
+                        new ConstantExpression(m.declaringClass.name),
+                        new ConstantExpression(m.name),
+                        new ConstantExpression(sourceUnit.name)
+                ))
+        ));
+        b = new MethodCallExpression(b, "withClosureType",
+                new TupleExpression(new ClassExpression(config.closureType)));
+
+        def tag = getTrustTag()
+        if (tag!=null) {
+            b = new MethodCallExpression(b, "contextualize",
+                    new PropertyExpression(new ClassExpression(ClassHelper.makeCached(tag)), "INSTANCE"));
+        }
+        return b;
+    }
+
+    /**
+     * {@link Trusted} or {@link Untrusted} tag that gets added to call site.
+     *
+     * @see 'doc/sandbox.md'
```

Comment:
Invalid syntax AFAIK; you can specify either a member (as per `link`), or an HTML `<a>` element.